src/utils/ADempiere/dictionary/process.js

```diff
@@ -151,6 +151,7 @@ export function generateProcess({
 
     fieldsList = fieldsList.concat(fieldsRangeList)
     // order range fields
+    console.log({ fieldsList })
```

Comment:
@elsiosanchez please remove this console.log

Reply:
[[95ebee3](https://github.com/solop-develop/frontend-core/pull/2060/commits/95ebee31ba8310075bb2f93da62a0008deccb6ab)](url)